priv/static/swagger/swagger.json

```diff
@@ -2789,12 +2789,213 @@
         ]
       }
     },
-    "/txs": {
+    "/txs/{scope_type}/{range}": {
+      "get": {
+        "deprecated": false,
+        "description": "Get a transactions from beginning or end of the chain. More [info](https://github.com/aeternity/ae_mdw#transaction-querying)",
```

Comment:
An update for the info link:
https://github.com/aeternity/ae_mdw#transactions

Reply:
Done